package.json

```diff
@@ -26,7 +26,7 @@
     "prettier": "1.19.1"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "REACT_APP_GIT_HASH=\"$(git rev-parse HEAD)\" react-scripts start",
```

Comment:
So after doing some reading about pulling in env vars, and understanding what you meant by how this could be tricky - I attempted a different strategy, using the Git Hash which has a similar property to the version in that it will change on deploy.

But I'm not sure if this is clever, or if by pulling the value here at the start script I've polluted things/done something that is not best practice.

Clever, or try again?